.github/workflows/esp32.yml

```diff
@@ -23,4 +23,4 @@ jobs:
       - uses: actions/checkout@v2
       - name: Test
         run: |
-          tensorflow/lite/micro/tools/ci_build/test_esp32.sh
+          docker run --rm -v `pwd`:/opt/tflite-micro espressif/idf:release-v4.3 /opt/tflite-micro/tensorflow/lite/micro/tools/ci_build/test_esp32.sh
```

Comment:
I copied this from the xtensa f1 workflow which seems to use a custom image assocated with the tflm bot.  Here because the espressif image is public it just works.  

The -v part is a volume mount that brings the current working directory (the root of the tflite-micro repo) into the image in the /opt/tflite-micro directory.

Then the arg is the specific file from within the image to run.